packages/server/src/services/bitquery/fetcher.ts

```diff
@@ -44,15 +44,10 @@ export default class BitqueryFetcher {
 
     static async getLastDayOHLC(baseTokenId: string, quoteTokenId: string): Promise<DexTrade> {
```

Comment:
this was essentially the same as getPeriodOHLC where start and end were 1 day apart